packages/teams-js/src/public/runtime.ts

```diff
@@ -351,6 +363,9 @@ export const versionAndPlatformAgnosticTeamsRuntimeConfig: Runtime = {
       },
       update: {},
     },
+    hostEntity: {
+      tab: {},
+    },
```

Comment:
Putting this in the `versionAndPlatformAgnosticTeamsRuntimeConfig` object means that any version of Teams that don't use the host SDK (this would be Teams V1 and all versions of Teams Mobile) will use this runtime object. Based on this change, you are saying that, for example, all versions of Teams Mobile supports this capability. I have to believe that's not true, since you are just adding it now (unless you've already added support for it into the Teams Mobile codebase and it's rolled out to production).

I'll make a change to add a comment to this structure to make its usage clearer.

Reply:
FYI I now have a PR out to explain the usage of these structures more clearly:
[https://github.com/OfficeDev/microsoft-teams-library-js/pull/2451](https://github.com/OfficeDev/microsoft-teams-library-js/pull/2451)

Reply:
Removed